docs/essentials/widget_language.md

```diff
@@ -527,30 +527,32 @@ We have already introduced `id` before. Here, we use it to name the object `gree
 >
 >```
 
-- `from` field accept a value of a property which the animate start from.
+- The `from` field accepts a value of a property where the animation should start with.
 
-Someone may need clarification, we have a `from` field describe animate come from, but why not have a `to` field describe where to go. In Ribir, animations are only visual effects and should not affect any application data. Ribir animate always finished at the value of the animate property, that's where the animation goes.
+Why do we have a `from` field to describe where to start the animation from, but not a `to` field to define where to end the?  
+In Ribir animations are only visual effects and should not affect any application data. Ribir animations always finish at the current value of the animation property. That's where the animation goes.
 
 > Tips
 >
-> Because animations are only visual effects, the animate property must be a field of `Render` widget.
+> Because animations are only visual effects, the animate property must be a field of the `Render` widget.
 
 We have learned how to declare an `Animate` object. The next step is to trigger it. We need to trigger it when the `greet` text is modified, so add this statement to `finally` block.
+
 ```rust
 let_watch!(greet.text)
   .subscribe(move |_| greet_enter.run());
 ```
 
-The animation work as we expected now. In practice, we use the predefined `Transition` in theme. We polish over animate code.
+The animation now works as we expected. In practice we use the predefined `Transition` from the theme. **We polished over the animation code.**(I don't understand what that means)
```

Comment:
Well, the last sentence is exactly a useless and misleading one. You can delete it directly.